Automattic/wp-calypso
# Jetpack Connect: Optimize site lists requests

Jetpack connect's authorization currently fetches the entire site list before AND after connecting.

This PR attempts to only fetch the single site we are connecting instead.

To test:
- get this PR going at calypso.localhost:3000
- run through the different connection scenarios at p2: P7rd6c-Me-p2 and ensure there are no regressions
- if you are using a wordpress.com account with a lot of sites, for example an automattic account, it should be much faster 🎉
